config/routes.rb

```diff
@@ -113,8 +113,13 @@
       # /api/v1/public/
       namespace :public, defaults: { format: :json } do
         get "channels", controller: "channels"
+        namespace :channels, defaults: { format: :json } do
```

Comment:
I think it might make more sense to have it like
```suggestion
        resources :channels, only: [:index] do
```
Then you can remove the route above it and it will be easier to add on more RESTlike routes later

Reply:
Good catch. I'll fix

Reply:
Actually I reverted this given that we'd have to attach a `id` in the curl request.